src/web/theme/modules/Header/Navigation/Navigation.js

```diff
@@ -2,17 +2,15 @@ import React from "react";
 import { CartModal } from "../../Cart";
 import Button from "theme/ui/atoms/Button";
 import IconWithLabel from "theme/ui/molecules/IconWithLabel";
+import "./Navigation.scss";
 
 const Navigation = () => {
   return (
-    <nav>
-      <CartModal>
-        {openCart => (
-          <Button type="invisible" onClick={openCart}>
-            <IconWithLabel icon="cart">Cart</IconWithLabel>
-          </Button>
-        )}
-      </CartModal>
+    <nav className="navigation">
+      <span>Browse sites</span>
```

Comment:
`<ul>` inside of the `<nav>` is more correct then spans here, also you have "Browse sites" twice ?